src/Abp.OnlineClientStore.Redis/RealTime/Redis/RedisKeyValueStore.cs

```diff
@@ -0,0 +1,90 @@
+﻿using System;
+using System.Collections.Immutable;
+using System.Linq;
+using JetBrains.Annotations;
+using StackExchange.Redis;
+
+namespace Abp.RealTime.Redis
+{
+    public class RedisKeyValueStore<TKey, TValue> : IRedisKeyValueStore<TKey, TValue>
```

Comment:
what is the reason of having this store?

Reply:
Just key value helper store around redis, if requared can be moved into client store

Reply:
Are the functionalities of this store same as `AbpRedisCache`?

Reply:
No, i need option to get all values which from what i see it's not implemented into AbpRedisCache, what do you think, extended AbpRedisCache? Is it good idea to use RedisCache for OnlineClient? There is stuff there which are not requared for storing and retriving online client, ExpireTime, localized key, abpcachedata etc..